packages/@glimmer/integration-tests/test/owner-test.ts

```diff
@@ -81,6 +169,34 @@ class OwnerTest extends RenderTest {
 
     assert.verifySteps(['foo-bar owner called', 'foo-baz owner called'], 'owners used correctly');
   }
+
+  @test
+  'owner can be changed by a component with the hasSubOwner capability'(assert: Assert) {
```

Comment:
This test is unrelated, I just realized we didn't have it (or at least not an obvious one for it) so wanted to add it